parser.y

```diff
@@ -2311,7 +2312,7 @@ NumLiteral:
 
 
 CreateIndexStmt:
-	"CREATE" CreateIndexStmtUnique "INDEX" IfNotExists Identifier IndexTypeOpt "ON" TableName '(' IndexColNameList ')' IndexOptionList LockClauseOpt
+	"CREATE" IndexKeyTypeOpt "INDEX" IfNotExists Identifier IndexTypeOpt "ON" TableName '(' IndexColNameList ')' IndexOptionList LockClauseOpt
```

Comment:
could you fix `algorithm_option` by the way?

Reply:
https://github.com/pingcap/parser/issues/429

Reply:
^_^ sure

Reply:
fix it in this pr